<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>322</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>楷体</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit_area">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>8</y>
      <width>97</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="city_lable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文新魏</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>城市：最近一次更新：</string>
    </property>
   </widget>
   <widget class="QLabel" name="datatime_label">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>00:00:00</string>
    </property>
   </widget>
   <widget class="QLabel" name="ico_label_0">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/pics/3d_180/99.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="ico_label_1">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>60</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/pics/3d_180/99.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="ico_label_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>60</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/pics/3d_180/99.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="day_label_0">
    <property name="geometry">
     <rect>
      <x>54</x>
      <y>130</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>N/A</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="day_label_1">
    <property name="geometry">
     <rect>
      <x>175</x>
      <y>130</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>N/A</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="day_label_2">
    <property name="geometry">
     <rect>
      <x>304</x>
      <y>130</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>N/A</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="hum_label_0">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>190</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>N/A</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="hum_label_1">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>190</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>N/A</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="hum_label_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>190</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>N/A</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="wind_label_0">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>190</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>N/A</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="wind_label_1">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>190</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>N/A</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="wind_label_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>190</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>N/A</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="temp_label_0">
    <property name="geometry">
     <rect>
      <x>58</x>
      <y>160</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>N/A</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="temp_label_1">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>160</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>N/A</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="temp_label_2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>160</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>N/A</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="save_city_pushButton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>10</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/bar_ico/bar_ico/site.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>ico_label_0</zorder>
   <zorder>lineEdit_area</zorder>
   <zorder>city_lable</zorder>
   <zorder>datatime_label</zorder>
   <zorder>ico_label_1</zorder>
   <zorder>ico_label_2</zorder>
   <zorder>day_label_0</zorder>
   <zorder>day_label_1</zorder>
   <zorder>day_label_2</zorder>
   <zorder>hum_label_0</zorder>
   <zorder>hum_label_1</zorder>
   <zorder>hum_label_2</zorder>
   <zorder>wind_label_0</zorder>
   <zorder>wind_label_1</zorder>
   <zorder>wind_label_2</zorder>
   <zorder>temp_label_0</zorder>
   <zorder>temp_label_1</zorder>
   <zorder>temp_label_2</zorder>
   <zorder>save_city_pushButton</zorder>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="tabletTracking">
    <bool>false</bool>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="movable">
    <bool>true</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::AllToolBarAreas</set>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="set_key_action"/>
   <addaction name="about_action"/>
   <addaction name="separator"/>
   <addaction name="exit_action"/>
  </widget>
  <action name="set_key_action">
   <property name="icon">
    <iconset resource="pic.qrc">
     <normaloff>:/bar_ico/bar_ico/set_key.png</normaloff>:/bar_ico/bar_ico/set_key.png</iconset>
   </property>
   <property name="text">
    <string>设置key</string>
   </property>
   <property name="toolTip">
    <string>设置key</string>
   </property>
  </action>
  <action name="exit_action">
   <property name="icon">
    <iconset resource="pic.qrc">
     <normaloff>:/bar_ico/bar_ico/exit.png</normaloff>:/bar_ico/bar_ico/exit.png</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出</string>
   </property>
  </action>
  <action name="about_action">
   <property name="icon">
    <iconset resource="pic.qrc">
     <normaloff>:/bar_ico/bar_ico/about.png</normaloff>:/bar_ico/bar_ico/about.png</iconset>
   </property>
   <property name="text">
    <string>关于</string>
   </property>
   <property name="toolTip">
    <string>关于</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="pic.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>save_city_pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>get_slot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>478</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>480</x>
     <y>91</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>about_slot()</slot>
  <slot>get_slot()</slot>
  <slot>set_key_slot()</slot>
  <slot>save_city()</slot>
 </slots>
</ui>
